---
sensor_msgs/PointCloud2 collected_cloud
